docs/plugins/create-a-plugin.md

```diff
@@ -11,13 +11,15 @@ A Backstage Plugin adds functionality to Backstage.
 To create a new plugin, make sure you've run `yarn install` and installed
 dependencies, then run the following on your command line (a shortcut to
 invoking the
-[`backstage-cli create-plugin`](../local-dev/cli-commands.md#create-plugin))
+[`backstage-cli new --select plugin`](../local-dev/cli-commands.md#new))
 from the root of your project.
 
 ```bash
 yarn new --select plugin
 ```
 
+[//]: # 'Todo: change the asses to show the output of the new command'
```

Comment:
This seems to be lingering still. Last thing remaining I think?

Reply:
Unfortunate typo 🤣
The output is shown as an image and I was not sure if I should so the output in my windows system. Hence I kept it as such. What do you think? If a powershell output is fine, I can screenshot that and replace the asset.

Reply:
Oh... Well OK, we can change the asseT separately and just be happy with fixing the typo for now :)

Reply:
@freben I updated the image. Let me know if it looks ok, If not, I will amend the commit and not update the image.